libs/wingsdk/src/std/map.ts

```diff
@@ -55,7 +65,7 @@ export class ImmutableMap {
   /**
    * Create an immutable shallow copy of this map
    *
-   * @macro Object.freeze(new Map($self$))
+   * @macro Object.freeze({...$self$})
```

Comment:
`$self$` could be a multi-factor expression and it's not emitted with parens. This is probably good just-in-case.

```suggestion
   * @macro Object.freeze({...($self$)})
```